contracts/staking/test/unit_tests/delegator_reward_test.ts

```diff
@@ -207,7 +207,7 @@ blockchainTests.resets('Delegator rewards unit tests', env => {
     }
 
     async function finalizePoolAsync(poolId: string): Promise<ResultWithTransfers<{}>> {
-        const receipt = await testContract.originalFinalizePool.awaitTransactionSuccessAsync(poolId);
```

Comment:
This change actually changes the semantics of the test on [this line](../blob/f55889325bb365b79234cb2657f4470551ef101f/contracts/staking/test/unit_tests/delegator_reward_test.ts#L761). But maybe we don't care about that test.

Reply:
Hmm, I may be able to preserve the original logic via a `delegatecall`. Had to make this change because `finalizePool` is now `external`.